freestyle/shared/src/test/scala/freestyle/implicits.scala

```diff
@@ -53,7 +53,7 @@ class implicitsTests extends WordSpec with Matchers {
 
     "provide a custom implicit not found message for a missing Capture instance" in {
       shapeless.test.illTyped(
-        """SCtors1[SCtors1.Op].x(1).exec[scala.util.Try]""",
+        """SCtors1[SCtors1.Op].x(1).freeS.exec[scala.util.Try]""",
```

Comment:
The test is about the conversion from `FreeS.Par[F, _]` or `FreeS[F, _]` to `Future` (which is what the `play` integration provides). I have added another _conversion_ to the main code, and a test for it.